IntoHistory/HeroListDescriptionCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  HeroListDescriptionCell.swift
+//  IntoHistory
+//
+//  Created by 김민택 on 2022/08/12.
+//
+
+import UIKit
+
+class HeroListDescriptionCell: UICollectionViewCell {
+    
+    // MARK: - Property
+    
+    static let identifier = "HeroListDescriptionCell"
+    
+    // MARK: - View
+    
+    let descriptionSection: UIStackView = {
```

Comment:
p1;

descriptionSection는 해당 class 내부에서만 사용하고 있기 때문에 접근제어자 추가 부탁드립니다.

```suggestion
    private let descriptionSection: UIStackView = {
```